README.md

```diff
@@ -51,6 +51,33 @@ export default defineNuxtConfig({
 
 That's it! You can start writing and rendering markdown files in your Nuxt project ✨
 
+### WebAssembly (WASM)
+
+For some Nuxt projects, depending on your build preset, you may need to adjust settings to allow WebAssembly (WASM) to be used for the default Shiki highlighter.
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  modules: ['@nuxtjs/mdc'],
+  $production: {
```

Comment:
Ideally, I'd like to see if it's possible to leave the `wasm` enabled for `$development` as well?

See the errors when running this reproduction's dev server: https://github.com/adamdehaven/mdc-wasm-monorepo